kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -286,6 +370,7 @@ log_level -i "ENVIRONMENT_NAME:                         $ENVIRONMENT_NAME"
 log_level -i "EXTERNAL_FQDN:                            $EXTERNAL_FQDN"
 log_level -i "STORAGE_PROFILE:                          $STORAGE_PROFILE"
 log_level -i "TENANT_ENDPOINT:                          $TENANT_ENDPOINT"
+log_level -i "K8S_AZURE_CLOUDPROVIDER_RELEASE:          $K8S_AZURE_CLOUDPROVIDER_RELEASE"
```

Comment:
Not a constant

Reply:
What do you mean ??

Reply:
never mind, my bad, I did not notice I had to expand the file.